### DISCUSSION TITLE
Redefine of ROG Ally joystick behavior mode

### DISCUSSION THREAD

**Discussion by gaowanliang** [seerge/g-helper#2452] [Q&A] [answered]
Currently in mouse mode, the left joystick has exactly the same behavioral pattern as the DPad, is it possible to remap the behavioral pattern of the joystick, for example mapping the left joystick to a mouse wheel scrolling up and down, left and right (left and right scrolling wheels are similar to the logitech master mouse, which I'm using)

**Comment by seerge** [comment, marked as answer]
@gaowanliang hello, 

In Mouse mode joystick emits keyboard arrow keys (up / down  / left / right).

It's impossible to bind it to mouse wheel unfortunately.

**Reply by gaowanliang** [reply]
sad 😢

**Reply by gaowanliang** [reply]
But one thing I'd like to change is the behavioral pattern of the joystick instead of the DPad

**Reply by seerge** [reply]
@gaowanliang sorry, I meant joystick emits arrow keys. 

You can bind joystick only to arrows or to mouse movements, that's it.
